lighthouse-core/gather/driver.js

```diff
@@ -1477,6 +1492,32 @@ class Driver {
     }
   }
 
+  /**
+   * @param {string} url
+   * @param {(LH.IcuMessage | string)[]} LighthouseRunWarnings
+   */
+  async getImportantStorageWarning(url, LighthouseRunWarnings) {
+    const usageData = await this.sendCommand('Storage.getUsageAndQuota', {
+      origin: url,
+    });
+    /** @type {Object.<string, string>} */
+    const storageTypeNames = {
+      local_storage: 'Local Storage',
+      indexeddb: 'IndexedDB',
+      websql: 'Web SQL',
+    };
+    const locations = usageData.usageBreakdown
+      .filter(usage => usage.usage)
+      .map(usage => storageTypeNames[usage.storageType] || '')
+      .filter(Boolean);
+    if (locations.length) {
+      LighthouseRunWarnings.push(str_(
+        UIStrings.warningData,
+        {locations: locations.join(', '), locationCount: locations.length}
```

Comment:
Since we've deferred on this in the past, unless anyone else wants to chime in on this string, maybe we can bring it up in the monday eng sync and make a decision on this then?

Reply:
I'll put it in the sync doc for now